src/Pagination/Pagination.svelte

```diff
@@ -136,6 +140,10 @@
         labelText="Page number, of {totalPages} pages"
         inline
         hideLabel
+        on:input="{() => console.log('select input event')}"
```

Comment:
This seems unintentional?

Reply:
Oops, must have left that in during testing by accident. It's now rmeoved.